src/Akeneo/Pim/Enrichment/Bundle/Twig/AttributeExtension.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Pim\Enrichment\Bundle\Twig;
+
+use Twig_Extension;
+use Twig_SimpleFunction;
+
+/**
+ * Twig extension to manage attribute from twig templates
+ *
+ * @author    JM Leroux <jean-marie.leroux@akeneo.com>
+ * @copyright 2020 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class AttributeExtension extends Twig_Extension
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getFunctions()
+    {
+        return [
+            new Twig_SimpleFunction('contains_asian_characters', [$this, 'containsAsianCharacters']),
+        ];
+    }
+
+    /**
+     * Test an attribute content for Asian characters
+     */
+    public function containsAsianCharacters(?string $text): bool
```

Comment:
```suggestion
    public function containsChineseCharacters(?string $text): bool
```
Asia has more than 2400 languages :slightly_smiling_face:

Reply:
It's not only Chinese, but I can be more precise with smth like `containsHanCharacters`